src/Jaeger/Senders/HttpSender.cs

```diff
@@ -45,15 +46,16 @@ private HttpSender(Builder builder)
             }
 
             _transport = new THttpClientTransport(collectorUri, customHeaders);
-            _agentClient = new Agent.Client(ProtocolFactory.GetProtocol(_transport));
+            _protocol = ProtocolFactory.GetProtocol(_transport);
         }
 
         protected override async Task SendAsync(ThriftProcess process, List<ThriftSpan> spans, CancellationToken cancellationToken)
         {
             try
             {
                 var batch = new ThriftBatch(process, spans);
-                await _agentClient.emitBatchAsync(batch, cancellationToken).ConfigureAwait(false);
+                await batch.WriteAsync(_protocol, cancellationToken);
```

Comment:
.ConfigureAwait(false) is missing on both await calls.

Reply:
Will add them in a second. Thanks!